tools/db_stress.cc

```diff
@@ -3506,31 +3506,48 @@ class AtomicFlushStressTest : public StressTest {
     }
     size_t num = column_families_.size();
     assert(num == iters.size());
+    std::vector<Status> statuses(num, Status::OK());
     do {
       size_t valid_cnt = 0;
+      size_t idx = 0;
       for (auto& iter : iters) {
         if (iter->Valid()) {
           ++valid_cnt;
+        } else {
+          statuses[idx] = iter->status();
         }
+        ++idx;
       }
       if (valid_cnt == 0) {
-        break;
-      } else if (valid_cnt != iters.size()) {
-        fprintf(stderr, "Finished iterating the following column families:\n");
         for (size_t i = 0; i != num; ++i) {
-          if (!iters[i]->Valid()) {
-            fprintf(stderr, "%s ", column_families_[i]->GetName().c_str());
+          const auto& s = statuses[i];
+          if (!s.ok()) {
+            fprintf(stderr, "Iterator on cf %s has error: %s\n",
+                    column_families_[i]->GetName().c_str(),
+                    s.ToString().c_str());
+          } else {
+            fprintf(stderr, "Finished scanning cf %s\n",
+                    column_families_[i]->GetName().c_str());
           }
         }
-        fprintf(stderr,
-                "\nThe following column families have data that have not been "
-                "scanned:\n");
+        break;
+      } else if (valid_cnt != iters.size()) {
         for (size_t i = 0; i != num; ++i) {
-          if (iters[i]->Valid()) {
-            fprintf(stderr, "%s ", column_families_[i]->GetName().c_str());
+          if (!iters[i]->Valid()) {
+            if (statuses[i].ok()) {
+              fprintf(stderr, "Finished scanning cf %s\n",
+                      column_families_[i]->GetName().c_str());
+            } else {
+              fprintf(stderr, "Iterator on cf %s has error: %s\n",
```

Comment:
should we be calling `SetVerificationFailure` in any of these cases?

Reply:
Done.